applications/web3-open-jurisdiction-stack.md

```diff
@@ -0,0 +1,346 @@
+
+
+- We developed and launched the Open Layer platform, which represents a pioneering game theory-based Online Dispute Resolution (ODR) system tailored for micro-claims;
+
+
+- Another significant achievement was the release of the Open Justice Platform, a comprehensive solution that digitises the UNCITRAL Model Law, offering an end-to-end framework for dispute resolution processes;
+
+
+- We have actively organised academic workshops and conducted research on topics such as jurisdiction, smart legal contracts, and ODR in collaboration with esteemed universities worldwide, including Radboud University, IE University, and Hull University;
+
+
+- Our collaborations and training activities extend to reputable arbitration institutions, such as the MCIA (Mumbai Center of International Arbitration);
+
+
+- We have had the privilege of interacting with numerous international arbitrators and judges across the globe, establishing valuable connections and fostering knowledge exchange;
+- We have participated in the Oxford University Deep Tech Dispute Resolution Lab;
+- We have engaged in collaborative efforts with the United Kingdom Jurisdiction Taskforce (UKJT) to develop digital dispute resolution rules;
+
+
+- We have participated in the leading legal tech acceleration program MDR Lab. This experience has allowed us to specifically focus on the area of digital arbitration.
+
+These milestones exemplify our team's extensive involvement in the field, demonstrating our deep commitment to advancing the realms of jurisdiction, ODR, and legal technology.
+
+**Intended venue for results publication and the timeline for publication.**
+
+**What your project is not or will not provide or implement**
+
+This research project is focused on conducting comprehensive investigations in the technical needs of the ecosystem, and as such, it will not involve a complete implementation of the open jurisdiction stack itself.
+
+The ultimate deliverable of this project will be a go/no-go decision regarding the jurisdictional requirements within the ecosystem. It will provide valuable insights and recommendations based on the findings, enabling stakeholders to make informed decisions regarding the implementation of a jurisdiction framework.
+
+We also aim to release at least one open source component of the stack to engage and share with the community around how the development of the jurisdiction stack could look like. This endeavour will be done also in case of “no-go” outcome from the research. Sentiment and needs towards a jurisdiction may change in the future and we think it would be wise at least to leave a tangible outcome from this endeavour from where anyone can resume.
```

Comment:
> This endeavour will be done also in case of “no-go” outcome from the research.

This sounds like a waste of resources tbh. Why would you implement something that doesn't work (if that was the case)?